var/spack/repos/builtin/packages/py-pygdal/package.py

```diff
@@ -29,5 +29,6 @@ class PyPygdal(PythonPackage):
     depends_on('py-numpy@1.0.0:', type=('build', 'run'))
     depends_on('gdal@3.0.1', type=('build', 'link', 'run'), when='@3.0.1.5')
     depends_on('gdal@2.4.2', type=('build', 'link', 'run'), when='@2.4.2.5')
+    depends_on('gdal@2.4.1', type=('build', 'link', 'run'), when='@2.4.0')
```

Comment:
This version is not in Spack

Reply:
It appears `gdal@2.4.1` has since been added.